### PR TITLE
Create Opay.json

### PR DESCRIPTION
**Pull request by emmanuellar**
Added the json file of the Opay privacy policy

### CODE REVIEW

File: Opay.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "OPay",
+  "documents": {
+    "Privacy Policy": {
+      "fetch": "https://www.opayweb.com/privacy-policy/",
+      "select": [
+        ".TCandPrivacyStyles__Content-sc-1tbbp80-0:nth-child(1)",
+        ".TCandPrivacyStyles__Content-sc-1tbbp80-0:nth-child(2)"
```
Comment (Cli4d) [inline Opay.json:8]: For better stability, it is preferable to use [simple CSS selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selectors_and_combinators#basic_selectors). If selecting specific parts of the webpage results in a complex selector like [pseudo selectors](https://www.w3schools.com/css/css_pseudo_elements.asp), you can select a larger area of the webpage and then remove the insignificant areas from it.

**Review by Cli4d — COMMENTED** [review]
Thank you for your contribution to Open Terms Archive. After reviewing it, I have identified some complex CSS selectors that can be replaced with simpler ones. I will make the necessary changes and merge your contribution. Thank you again for your time.

**Review by Cli4d — APPROVED** [review]
✅

**Comment by MattiSG** [comment]
There was a mistake in this addition that should have been caught in review: the declared service name is `OPay`, but the file name (i.e. service ID) is`Opay` (no capital on the P). Which one is the right one?

**Comment by MattiSG** [comment]
After looking into the terms, I opened https://github.com/OpenTermsArchive/contrib-declarations/pull/959 to correct the service ID 🙂

**Pull request merged**